pontomais/clockin.py

```diff
@@ -1,9 +1,18 @@
 # ttl.py - Calculates when to finish your workday based on data from Pontomais
 # Setup: insert auth in function request() with the instructions there and run.
 import json
+import sys
 from datetime import datetime
 from uuid import uuid4
 
+try:  
+    if sys.version_info[0] > 2: # Python 3
+        from http.client import responses
+    else: # Python 2
+        from httplib import responses 
+except ImportError:
+    pass # explicitly ignoring
```

Comment:
```suggestion
from http.client import responses
```

Python 2 is not supported and deprecated, and this project does use things of python 3, therefore you don't really need this.